### ISSUE TITLE
Minimize to system tray option (Windows)

### ISSUE THREAD

**Issue by NodusCursorius** [EDCD/EDMarketConnector#126]
Would love to see the window management options extend to allow minimizing the primary EDMC window to the System Tray (systray) instead of the Task Bar, please. This would also, naturally, remove it from the alt-tab listing since the program can already be interacted with via your hotkey feature.

**Comment by bodylinksystems** [comment]
I'm not sure about the intended hotkey behavior. I assigned it to INS key and I don't see anything happening unless I actually interact with the app on my other monitor.  Do I even need to interact with it since it's going to update automatically anyway? Is the hotkey to start the app or what?

**Comment by JoelleEmmily** [comment]
I would very much like to see this feature added.

**Comment by inorton** [comment]
@bodylinksystems when you dock the market update/edsm stuff is usually automatic unless you've turned that off in the _Output_ tab

**Comment by j33p4meplz** [comment]
Would LOVE this feature added.

**Comment by AndreGeddert** [comment]
This one is a year old. I would also like to see this feature. Any Plans, developsers?

**Comment by inorton** [comment]
Hmm, I might have a go, I've never really "needed" this feature myself

**Comment by AndreGeddert** [comment]
i would try it myself, but im very new to python. I would need hours or days. For an experienced python hacker this might be minutes of work :)

**Comment by inorton** [comment]
http://www.brunningonline.net/simon/blog/archives/SysTrayIcon.py.html  looks useful

**Comment by inorton** [comment]
I've spent a few mins trying to get it to work without much luck.

**Comment by inorton** [comment]
I've used http://rbtray.sourceforge.net/ in the past, that might be useful

**Comment by inorton** [comment]
@AndreGeddert yup, rbtray looks actually pretty cool, you just right click on the minimise button, works with every app!

**Comment by AndreGeddert** [comment]
Just tried it. It works with many apps like putty or even windows file explorer, but dont seem to work with apps that have a customized minimise button like EDMC, the Oculus Rift Software and my DAW. All of the three have a custom minimise button. But thanks for the hint anyways.

**Comment by Leonick** [comment]
@AndreGeddert If you set EDMC to the default theme it has standard window controls and rbtray does work.

I'd much prefer if EDMC had this as an option instead of having to run a background app that somehow hooks into the system to allow it though.

**Comment by mIRCon** [comment]
I would also like minimize to system tray feature @Athanasius :)

**Comment by Seforian** [comment]
I would love this feature too, in the addition. Check box:
1. Start minimized system tray.
2. Start minimized when system starts.

**Comment by SweetyVolty** [comment]
win+R : shell:startup 

enter 

place a shortcut and done. 

It will start every single startup till deleted or disabled at task manager. 

And Athanasius sorry about my request, I didn't saw this one before posting mine and also thank you for maintaining the edmc <3

**Comment by Athanasius** [comment]
Note to self, this might do the job https://github.com/Infinidat/infi.systray

**Comment by A-UNDERSCORE-D** [comment]
May also be doable in plain tk according to https://stackoverflow.com/questions/56109891/minimize-the-window-tkinter-in-the-windows-system-tray

**Comment by SayakMukhopadhyay** [comment]
> 
> 
> May also be doable in plain tk according to https://stackoverflow.com/questions/56109891/minimize-the-window-tkinter-in-the-windows-system-tray

I checked, the answer in that question is incorrect and is just telling the user to minimize to taskbar and not the system tray. Seems like many non windows people in that question don't know the distinction.

**Comment by Athanasius** [comment]
An implementation of this has now been merged into `develop` and will be in the next 5.0.0 pre-release (probably -beta7, as we need to do more Odyssey work once Alpha Phase 4 is up).

As requested, this is *minimise* to systray, and you'll need to set the new option on the 'Appearance' settings tab for it to do that, else current behaviour remains.  'Quit' from the systray (which will always be present) will close the entire program.  'Open' will restore the EDMarketConnector window if it's been minimised.

**Comment by Athanasius** [comment]
Unforunately we've now hit a snag with packaging infi.systray (actually one of its dependencies) with py2exe, so have had to comment out all of this code so as to get 5.0.0-beta7 built.

We'll report this upstream to py2exe and otherwise see if we can solve this for a future (pre-)release.

**Comment by Athanasius** [comment]
This is now only waiting on py2exe making a new release with the additional hooks in place to allow use of infi.systray.

**Comment by Athanasius** [comment]
No available in https://github.com/EDCD/EDMarketConnector/releases/tag/Release%2F5.2.2
